packages/NekoVM/build.sh

```diff
@@ -0,0 +1,20 @@
+TERMUX_PKG_HOMEPAGE="https://nekovm.org/"
+TERMUX_PKG_DESCRIPTION="The Neko Virtual Machine"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2-3-0
```

Comment:
This value cannot be used as version. Specify `2.3.0` there and do conversion in `TERMUX_PKG_SRCURL`.